test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -212,6 +212,11 @@
     numPodsToSchedule: 2000
 - template:
     desc: MixedSchedulingBasePod
+    nodes:
+      nodeTemplatePath: config/node-default.yaml
+      labelNodePrepareStrategy:
```

Comment:
adding labelNodePrepareStrategy to MixedSchedulingBasePod so the test can complete makes sense. if we want to merge this in 1.19, should we leave the other bits alone for now (continue using `failure-domain.beta.kubernetes.io/zone`, just have a single `zone1` value in SchedulingNodeAffinity, etc)?

Reply:
Done.